src/components/organisms/NewResourceWizard/NewResourceWizard.tsx

```diff
@@ -27,12 +31,80 @@ import {openNamespaceTopic, openUniqueObjectNameTopic} from '@utils/shell';
 
 import {getResourceKindHandler} from '@src/kindhandlers';
 
-import {SaveDestinationWrapper, StyledSelect} from './NewResourceWizard.styled';
+import {FileCategoryLabel, FileNameLabel, SaveDestinationWrapper, StyledSelect} from './NewResourceWizard.styled';
 
 const SELECT_OPTION_NONE = '<none>';
 
 const {Option, OptGroup} = Select;
 
+const fetchFullFileName = (filename: string, suffix?: string) => {
+  return `${filename}${suffix || ''}.yaml`;
+};
+
+const createFullFileName = (
+  subfiles: fs.Dirent[],
+  resourceName: K8sResource['name'],
+  resourceKind: K8sResource['kind'],
+  selectedFolder: string,
+  fileMap: FileMapType,
+  suffix: number,
+  includeKind?: boolean
+): string => {
+  const name = resourceName;
+  const nameKind = includeKind ? `-${resourceKind.toLowerCase()}` : '';
+  const nameSuffix = suffix ? ` (${suffix})` : '';
+  const fullFileName = fetchFullFileName(`${name}${nameKind}${nameSuffix}`);
+  let foundFile: fs.Dirent | FileEntry | undefined;
+
+  if (subfiles.length) {
+    foundFile = subfiles.find(dirent => dirent.name === fullFileName);
```

Comment:
We don't need to use the subfiles in here as we are saving just in our local folder, not outside the project, hence not needed to search for subfiles.

Reply:
This has been amended